mob.go

```diff
@@ -551,6 +566,11 @@ func start(configuration config.Configuration) error {
 
 	createRemoteBranch(configuration, currentBaseBranch)
 
+	if !currentBaseBranch.hasLocalBranch(localBranches) {
+		silentgit("checkout", "-b", currentBaseBranch.Name)
+		silentgit("checkout", currentBranch.Name)
```

Comment:
why do we need this line? It feels wrong

Reply:
Hi @simonharrer, first I thought we need it, because the git command which is used to check if there are unpushed commits. But because of your comment I thought more about it and if there is no local base branch there can not be unpushed commits. but after implementing this change the test failed again. In the function `sayLastCommitsList` we look at the local basebranch. I now implemented it the way that we try this behavior if it fails we try to check against remote. Let me know if you think it is better to directly compare to the remote base branch instead of trying to compare to the local branch first. I think it could make sense but was not completely sure about it.